core/api/src/config.rs

```diff
@@ -6,6 +6,15 @@ pub struct GraphQLConfig {
 
     pub graphql_uri:  String,
     pub graphiql_uri: String,
+
+    // Set number of workers to start.
+    // By default http server uses number of available logical cpu as threads count.
+    pub workers: usize,
+
+    // Sets the maximum per-worker number of concurrent connections.
+    // All socket listeners will stop accepting connections when this limit is reached for each
+    // worker. By default max connections is set to a 25k.
+    pub maxconn: usize,
```

Comment:
A socket will occupy a file descriptor, so it will failed at a unadjusted system which the default fd count is 1024.

Reply:
The default of 1024 is not sufficient and will need to be adjusted to a larger value in most scenarios.